src/components/ScholarshipDetailCardCell.test.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import ScholarshipDetailCardCell from './ScholarshipDetailCardCell';
+
+test('renders ScholarshipDetailCardCell', () => {
```

Comment:
Better would be a test for ScholarshipDetailCard that checks all the scholarship details show up. Maybe also an additional test for rendering a barely filled out Scholarship.